samcli/hook_packages/terraform/hooks/prepare.py

```diff
@@ -594,6 +727,7 @@ def _enrich_resources_and_generate_makefile(
     resources_types_enrichment_functions = {
         "ZIP_LAMBDA_FUNCTION": _enrich_zip_lambda_function,
         "IMAGE_LAMBDA_FUNCTION": _enrich_image_lambda_function,
+        "LAMBDA_LAYER": _enrich_lambda_layer,
```

Comment:
can you add a unit test case for testing enriching the lambda layers like this one https://github.com/aws/aws-sam-cli/blob/8c8bc2e54ba1a03b5bc6d0a4cf7bb8d67ea8d3e3/tests/unit/hook_packages/terraform/test_prepare_hook.py#L1360